### PR TITLE
docs(interaction): add update scroll speed ratio example

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🔧 Chore

- [x] Docs / demos update

### 📝 Description

增加滚动速率demo

### 🖼️ Screenshot

Kapture 2021-11-05 at 17 00 22

### CODE REVIEW

**Review by zxc0328 — APPROVED** [review]
LG

File: s2-site/examples/interaction/advanced/demo/scroll-speed-ratio.ts [hunk 1]
```diff
@@ -1,5 +1,32 @@
-import { S2Event, TableSheet } from '@antv/s2';
+import { TableSheet } from '@antv/s2';
 import '@antv/s2/dist/s2.min.css';
+import insertCss from 'insert-css';
+
+const defaultScrollSpeedRatio = 1;
+
+function createSlider(s2) {
```
Comment (wjgogogo) [inline s2-site/examples/interaction/advanced/demo/scroll-speed-ratio.ts:7]: slider放到头部吧，方便操作一点

Comment (xingwanying) [reply s2-site/examples/interaction/advanced/demo/scroll-speed-ratio.ts:7]: 雀食

Comment (lijinke666) [reply s2-site/examples/interaction/advanced/demo/scroll-speed-ratio.ts:7]: done

**Review by xingwanying — APPROVED** [review]
LG

**Pull request merged**